OpenTrans.net/Feature.cs

```diff
@@ -62,5 +67,10 @@ public class Feature
         /// Definition of the feature
         /// </summary>
         public string Template { get; set; }
+
+        /// <summary>
+        /// Information on the valid value range
+        /// </summary>
+        public string ValueType { get; set; }
```

Comment:
how about making this an enum? According to the docs, there are only three valid values